src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,15 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Represents a basic Command.
+ */
+public interface Command {
+    public String execute(TaskList list, Ui ui, Storage storage) throws DukeException;
+
+    public boolean isExit();
```

Comment:
One way to make the code cleaner is to have a default impl for isExit, 
```java
    public boolean isExit() {
        return false;
    }
```

Since most commands don't exit by default, and we can override this in the ExitCommand to return true